secvarctl.c

```diff
@@ -123,7 +123,7 @@ static struct backend *getBackend()
 		prlog(PR_WARNING, "WARNING!! Could not extract data from %s , assuming platform does not support secure variables\n", secVarFormatLocation);
 		goto out;
 	}
-	//loop through all known backends
+	// loop through all known backends
```

Comment:
This commit does not need to be in the history. If you want to keep this comment change, change the description instead to reflect what it is doing.